src/pages/Landing/index.tsx

```diff
@@ -172,18 +171,6 @@ export default function Landing() {
 
   const landingPageFlag = useLandingPageFlag()
```

Comment:
should we remove this now?

Reply:
i have a pr for it

Reply:
this default to enabled for now, and i have a diff pr to remove completely